src/cli/self_update.rs

```diff
@@ -74,6 +74,15 @@ Cargo's bin directory, located at:
 
     {cargo_home_bin}
 
+This can be modified with the CARGO_HOME envorinment variable.
+
+Rustup metadata and toolchains will be installed the Rustup home
```

Comment:
Grammar error here - "will be installed the" -> "will be installed into the"

Reply:
Fixed.  Also fixed misspelling of *environment*.